Hunting Queries/AuditLogs/RareAuditActivityByUser.yaml

```diff
@@ -19,33 +19,34 @@ query: |
   let current = 1d;
   let auditLookback = 14d;
   let AuditTrail = AuditLogs 
```

Comment:
Same as first about adding Global AllowList.

Reply:
Can only do the propertyName as there is no App information as this is done by a person.